exception-handling/raising-and-handling-exceptions.md

```diff
@@ -0,0 +1,118 @@
+# Exception Handling
+
+## Learning Goals
```

Comment:
*Reminder* for me tie in the sections of this lesson as right now it seems to jump around a bit. Things that would be great to add: 

- Providing real world examples of handling exceptions
- what it means to crash a program and why we want to prevent that,
- add questions.